plugins/docker/docker.plugin.zsh

```diff
@@ -28,7 +28,11 @@ alias dstp='docker container stop'
 alias dtop='docker top'
 alias dvi='docker volume inspect'
 alias dvls='docker volume ls'
+alias dcprune="docker container prune"
+alias diprune="docker image prune"
+alias dnprune="docker network prune"
 alias dvprune='docker volume prune'
+alias dprune="dcprune; diprune; dnprune; dvprune"
```

Comment:
@augustfengd Wouldn't `docker system prune -a` be more appropriate here?

Reply:
ref: https://docs.docker.com/reference/cli/docker/system/prune/

Reply:
Yes! This makes sense. I've updated my PR to do that.

Reply:
I also grouped the aliases by command so the diff is a bit noisy now.